controller/src/main/java/org/jboss/as/controller/SimpleAttributeDefinition.java

```diff
@@ -61,10 +60,8 @@
      * @param allowNull {@code true} if {@link org.jboss.dmr.ModelType#UNDEFINED} is a valid type for the value
      */
     public SimpleAttributeDefinition(final String name, final ModelType type, final boolean allowNull) {
-        super(name, name, null, type, allowNull, false, (MeasurementUnit) null,
-                (ParameterCorrector) null, (ParameterValidator) null, true, (String[]) null, (String[]) null,
-                (AttributeMarshaller) null, false, (DeprecationData) null, (AccessConstraintDefinition[]) null,
-                (Boolean) null, (AttributeParser) null);
+        super(name, name, null, type, allowNull, false, null, null, null, true, null, null,
+                null, false, null, null, null, null, null);
```

Comment:
The casts were there as a lame form of code comment because I was getting lost trying to figure out the param mapping. :) I figured the compiler would eliminate them anyway.

This would have been better I suppose:

super(name, name, null, type, allowNull, false, /*MeasurementUnit*/ null,

Anyway, I don't mind them going away.